masterlist.yaml

```
@@ -23821,3 +23821,11 @@ plugins:
     clean:
       - crc: 0x2358D3DD
         util: 'SSEEdit v4.0.3h (Hotfix 1)'
+
+  - name: 'TrissMerigoldW3.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/54724?/' ]
```

That `?` doesn't need to be there.

```suggestion
    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/54724/' ]
```